### ISSUE TITLE
Please add filtering by license

### ISSUE THREAD

**Issue by caryoscelus** [SvDvorak/BandcampMultitagSearch#12]
Since there's only limited number of allowed licenses on BC, the interface can be implemented as checkboxes (one might want to search for tracks under a few licenses, e.g. "CC-BY or CC-BY-SA" or "any CC license")

**Comment by SvDvorak** [comment]
I would probably do a dropdown list, possibly as part of an "Advanced Search" option. But currently development of CampExplorer is on ice (I don't have the time really) so we'll see if it gets implemented in the future.

**Comment by caryoscelus** [comment]
Thanks for quick reply! I understand that development isn't very active these days, but would you have enough time to review and merge the feature if someone implements it?

**Comment by SvDvorak** [comment]
Absolutely, I'm open for pull requests! Remember though that any new backend changes must be covered by new or updated code tests (see backend/spec).
